crates/usvg/src/text/flatten.rs

```diff
@@ -126,8 +220,129 @@ impl DatabaseExt for Database {
             let mut builder = PathBuilder {
                 builder: tiny_skia_path::PathBuilder::new(),
             };
+
             font.outline_glyph(glyph_id, &mut builder)?;
             builder.builder.finish()
         })?
     }
+
+    fn raster(&self, id: ID, glyph_id: GlyphId) -> Option<BitmapImage> {
+        self.with_face_data(id, |data, face_index| -> Option<BitmapImage> {
+            let font = ttf_parser::Face::parse(data, face_index).ok()?;
+            let image = font.glyph_raster_image(glyph_id, u16::MAX)?;
+
+            if image.format == RasterImageFormat::PNG {
+                let bitmap_image = BitmapImage {
+                    image: Image {
+                        id: String::new(),
+                        visibility: Visibility::Visible,
+                        size: Size::from_wh(image.width as f32, image.height as f32)?,
+                        rendering_mode: ImageRendering::OptimizeQuality,
+                        kind: ImageKind::PNG(Arc::new(image.data.into())),
+                        abs_transform: Transform::default(),
+                        // TODO: What to change to?
```

Comment:
Just an `image` size at 0x0 should be fine, afaik. After all, the flattened text has the identity transform now.